internal/characters/xingqiu/xingqiu.go

```diff
@@ -38,7 +37,15 @@ func NewChar(s *core.Core, w *character.CharWrapper, _ profile.CharacterProfile)
 
 func (c *char) Init() error {
 	c.a4()
-	c.naHook = common.NewNAHook(c.CharWrapper, c.Core, "xingqiu burst", burstKey, 60, burstICDKey, common.Get5PercentN0Delay, c.summonSwordWave)
-	c.naHook.NAStateHook()
+	common.NewNAHook(
+		c.CharWrapper,
+		c.Core,
+		"xingqiu burst",
+		burstKey,
+		60,
+		burstICDKey,
+		common.Get5PercentN0Delay,
+		c.summonSwordWave).
+		NAStateHook()
```

Comment:
super nit: does `gofmt` do this? IMO `.` should always start the next line when you are splitting between multiple (but also go syntax weird so whatever lol)